### PR TITLE
Create meta helper

### PR DESCRIPTION
**Pull request by martinbean**
Added a helper to create simple key–value pair meta tags. Also created a corresponding test.

### CODE REVIEW

**Comment by adamgoose** [comment]
Can you please add the ability to pass a third argument `array $attributes = []` that can be merged with the `compact('name', 'content')`? This will make it easier to install Facebook OpenGraph Meta Tags, which do not use the conventional name/content attributes (dumb, right?).

**Comment by martinbean** [comment]
@adamgoose Sure. Neither does schema.org mark-up, which use `itemprop` and `content` as well ;)

**Comment by martinbean** [comment]
@adamgoose Added `$attributes` as an optional third parameter and squashed the commit. Also added an additional test to cover creating an Open Graph meta tag.

**Comment by adamgoose** [comment]
Sorry, but it should use `array_merge()`. You'd be surprised how many people will put a `name` or `content` key in the `array $attributes`. Not sure how the + syntax will handle that.

**Comment by martinbean** [comment]
@adamgoose Oh. I took that approach from one of the other methods (not sure which one as on phone).

I’ll take a look later this evening or tomorrow morning as I’m travelling home, and heading straight out for the evening.

**Comment by martinbean** [comment]
@adamgoose Just pushed an update using `array_merge`. Let me know if that’s more what you’re after.

**Comment by RobinMalfait** [comment]
@martinbean failing tests :-1:

**Comment by martinbean** [comment]
@RobinMalfait Fixed.

File: tests/HtmlBuilderTest.php [hunk 1]
```diff
@@ -10,36 +10,49 @@
...
-
-    $this->assertEquals('<dl class="example"><dt>foo</dt><dd>bar</dd><dt>bing</dt><dd>baz</dd></dl>', $result);
-  }
+	/**
+	 * Setup the test environment.
+	 */
+	public function setUp()
+	{
+		$this->urlGenerator = new UrlGenerator(new RouteCollection, Request::create('/foo', 'GET'));
+		$this->htmlBuilder	= new HtmlBuilder($this->urlGenerator);
```
Comment (JosephSilber) [inline tests/HtmlBuilderTest.php]: This should be two spaces.

Comment (martinbean) [inline tests/HtmlBuilderTest.php]: @JosephSilber Not sure where you mean. Do you mean in between `$this->htmlBuilder` and HtmlBuilder($this->urlGenerator);`= new`?

Comment (tshafer) [inline tests/HtmlBuilderTest.php]: @martinbean Yes, please simply clean up the formatting on this.

**Comment by martinbean** [comment]
@JosephSilber @tshafer Indentation in test fixed. Sorry it took so long to address it, I was away for a few days last week.

**Comment by tshafer** [comment]
Thanks!

**Comment by martinbean** [comment]
@tshafer My pleasure!

**Pull request merged**